style.css

```diff
@@ -15,6 +15,28 @@ body {
   height: fit-content;
 }
 
+.hide {
+  display: none;
+}
+
+#toolbar {
+  max-width: 80vw;
+  min-width: fit-content;
+  height: fit-content;
+  margin-left: 30vw;
+}
+
+#toolbar li a {
+  list-style-type: none;
+  font-family: "Poppins", Arial, Helvetica, sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 15px;
+  line-height: 20px;
+  color: #344563;
+  text-decoration: none;
+}
+
```

Comment:
- [ ] Good job implementing the JavaScript functionality however, you design for mobile menu needs to be similar to the [Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1) template . Kindly implement the design before submitting your project for another review. cheers 👍 

The design should look like this             |  Your Design
:-------------------------:|:-------------------------:
![My Portfolio](https://user-images.githubusercontent.com/60618877/179644084-af9d2a23-5602-4a31-92e3-1fbad771ebde.gif)  |  ![Inaki's Portfolio_ Web developer](https://user-images.githubusercontent.com/60618877/179644528-6e90827e-3489-432f-b6f4-19e834767056.gif)